vividus-plugin-json/build.gradle

```diff
@@ -0,0 +1,10 @@
+project.description = 'Vividus plugin for JSON testing'
+
+dependencies {
+    implementation project(':vividus-bdd-engine')
+    implementation (group: 'com.flipkart.zjsonpatch', name: 'zjsonpatch', version: '0.4.11')
+
+    testImplementation platform(group: 'org.junit', name: 'junit-bom', version: versions.junit)
+    testImplementation(group: 'org.junit.jupiter', name: 'junit-jupiter')
+    testImplementation(group: 'org.mockito', name: 'mockito-junit-jupiter', version: versions.mockito)
+}
```

Comment:
also please add these 2 lines here at the end of file:
https://github.com/vividus-framework/vividus/blob/057c1f41cff1b2038b1979bde91b126f0da6e3f6/vividus-plugin-websocket/build.gradle#L16-L17